omero/sysadmins/server-upgrade.txt

```diff
@@ -131,6 +131,38 @@ below. Please refer to each section for additional details.
     :local:
     :depth: 1
 
+Check ahead for upgrade issues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There *may* be a "precheck" SQL script provided that performs various
```

Comment:
An alternative here might to be to *always* provide a `precheck` script from 5.3.0 and onwards but have the default script be a no-op.